dask/dataframe/io/json.py

```diff
@@ -74,7 +77,7 @@ def to_json(
         for outfile, d in zip(outfiles, df.to_delayed())
     ]
     if compute:
-        dask.compute(parts)
+        dask.compute(parts, **compute_kwargs)
```

Comment:
If `compute_kwargs` is still `None` here then the `**` unpacking will fail, so we'll need a check above this to make it an empty-dict if it hasn't been specified (or to not pass it if it's `None`)